src/PKSim.Core/Services/SimulationRunner.cs

```diff
@@ -71,8 +71,26 @@ public void StopSimulation()
          else
             updatePersistableFromSettings(simulation);
 
+         updateSolverSettings(simulation);
          await simulationEngine.RunAsync(simulation, _simulationRunOptions);
          simulation.HasChanged = true;
       }
+
+      private void updateSolverSettings<TSimulation>(TSimulation simulation) where TSimulation : Simulation
```

Comment:
it goes deep into the simulation runner., Default is AsIs and it should not change anything